CRITICAL - 2025-02-07 20:18:32 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:18:32 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:19:01 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:19:01 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:19:39 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:19:39 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:19:41 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:19:41 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:19:41 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:19:41 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:19:42 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:19:42 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:19:42 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:19:42 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:22:44 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:22:44 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:31:12 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:31:12 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:32:08 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:32:08 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:32:22 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:33:40 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:33:40 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2025-02-07 20:37:44 --> Error: Class "Locale" not found
in SYSTEMPATH\CodeIgniter.php on line 218.
 1 FCPATH\index.php(67): CodeIgniter\CodeIgniter->initialize()
CRITICAL - 2025-02-07 20:37:44 --> ErrorException: Uncaught Error: Class "Locale" not found in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php:137
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Helpers\number_helper.php(56): format_number(6.0, 2, NULL, Array)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(152): number_to_size(6.0, 2)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Views\errors\html\error_exception.php(390): CodeIgniter\Debug\BaseExceptionHandler::describeMemory(6291456)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(258): include('C:\\xampp_php_8....')
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\BaseExceptionHandler.php(261): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\ExceptionHandler.php(112): CodeIgniter\Debug\BaseExceptionHandler->render(Object(Error), 500, 'C:\\xampp_php_8....')
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Debug\Exceptions.php(156): CodeIgniter\Debug\ExceptionHandler->handle(Object(Error), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#7 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(Error))
#8 {main}
  thrown
【Previous Exception】
Error
Class "Locale" not found
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(67): CodeIgniter\CodeIgniter->initialize()
#1 {main}
in SYSTEMPATH\Helpers\number_helper.php on line 137.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2025-02-07 19:39:44 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'deleonconstrucc...', 3306, '', 0)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Controllers\Home.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Home->index()
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Controllers\Home.php(21): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Home->index()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2025-02-07 19:39:44 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_estado`', [], false)
 3 APPPATH\Controllers\Home.php(21): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-02-07 21:09:15 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'deleonconstrucc...', 3306, '', 0)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Controllers\Home.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Home->index()
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Controllers\Home.php(21): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Home->index()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2025-02-07 21:09:15 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_estado`', [], false)
 3 APPPATH\Controllers\Home.php(21): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-02-07 21:09:22 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'deleonconstrucc...', 3306, '', 0)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Controllers\Home.php(21): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Home->index()
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\app\Controllers\Home.php(21): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(943): App\Controllers\Home->index()
#5 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\vendor\codeigniter4\framework\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp_php_8.2.12\htdocs\deleonconstruccion\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2025-02-07 21:09:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `c_estado`', [], false)
 3 APPPATH\Controllers\Home.php(21): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(943): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(503): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
